packages/domain-search/src/components/search-notice/index.tsx

```diff
@@ -30,6 +30,17 @@ export const SearchNotice = () => {
 			return null;
 		}
 
+		// For odmains that are transferrable (i.e. registered elsewhere and can be transferred to us), we want to show
```

Comment:
Do we want to add this new if block? I was thinking of creating a new static array, like `AVAILABLE_DOMAIN_STATUSES`, and then doing the check within the first if.

Reply:
Yep, that makes sense. I'll update the code